civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -5537,50 +5537,59 @@ function civihr_employee_portal_hrreport_custom_printtable($reportName, $dateFil
  * @param $form_state
  */
 function civihr_employee_portal_form_views_exposed_form_alter(&$form, &$form_state, $form_id) {
-  /**
-   *  Staff Directory filters form, HR Documents filters form
-   */
-  $exposed_filters_forms = ['views-exposed-form-civihr-staff-directory-page', 'views-exposed-form-hr-documents-hr-resources'];
+  switch ($form['#id']) {
+    // People's report filters:
+    case 'views-exposed-form-civihr-report-people-default':
+      _civihr_employee_portal_form_view_exposed_form_alter_people_report($form, $form_state, $form_id);
+      break;
+    // Leave and Absence's report filters:
+    case 'views-exposed-form-civihr-report-leave-and-absence-default':
+      _civihr_employee_portal_form_view_exposed_form_alter_leave_report($form, $form_state, $form_id);
+      break;
+    // Staff Directory and HR Document's form filters:
+    case 'views-exposed-form-civihr-staff-directory-page':
+    case 'views-exposed-form-hr-documents-hr-resources':
+      _civihr_employee_portal_form_view_exposed_form_alter_actions_wrapper($form, $form_state, $form_id);
+      _civihr_employee_portal_form_view_exposed_form_alter_labels_as_placeholders($form, $form_state, $form_id);
+      break;
+  }
 
-  if (isset($form['#id']) && in_array($form['#id'], $exposed_filters_forms)) {
-    $form['#attributes']['class'][] = 'row';
+  $form['submit']['#value'] = 'Filter';
+}
 
-    $form['submit']['#attributes'] = ['class' => ['chr_action']];
-    $form['submit']['#prefix'] = '<div class="col-sm-12"><div class="chr_panel__footer"><div class="chr_actions-wrapper">';
-    $form['submit']['#suffix'] = '</div></div></div>';
+function _civihr_employee_portal_form_view_exposed_form_alter_people_report(&$form, $form_state, $form_id) {
```

Comment:
Since `$form_state` is passed by reference to the original function, so it's better to do the same for these functions you've extracted in case we need to change the form state in any of them